src/angular-app/bellows/apps/activity/activity-container.component.html

```diff
@@ -52,7 +52,7 @@
                     </div>
                     <button class="btn btn-sm brn-default" data-ng-click="$ctrl.filterDateOpen = !$ctrl.filterDateOpen">Close <i class="fa fa-times"></i></button>
                 </div>
-            </div>
+            </div> -->
```

Comment:
Good that you commented this out instead of removing it, as that will help us remember to come back and add a replacement date picker at some point. Could you add a comment to that effect (like "TODO: Add a date picker that doesn't use moment.js") so we'll know why this is commented out in the future, when we've forgotten about this PR?